client/js/bundles/document/elementsAdminList.js

```diff
@@ -11,7 +11,7 @@
  * This is the entrypoint for elements_admin_list.html.twig
  */
 
-import AddonWrapper from '@DpJs/components/addon/AddonWrapper'
+import AddonWrapper from '@demos-europe/demosplan-ui'
```

Comment:
Now, AddonWrapper is imported from two places. here, from demosplan-ui, and in administrationDashboard.js from core.

Reply:
we decided to not put the addonwrapper into demosplan-ui hence the path has been adjusted here: 
https://github.com/demos-europe/demosplan-core/pull/1486/commits/0e5c334bf1045388cdc948a51732f401fe8525d3